Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP_test.py

```diff
@@ -0,0 +1,1267 @@
+    requests_mock.delete(
+        f'{BASE_URL}/computers/{args["connector_guid"]}/isolation',
+        json=mock_response
+    )
+
+    from CiscoAMP import computer_isolation_delete_command
+    response = computer_isolation_delete_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.ComputerIsolation'
+    assert response.outputs[0]['available'] == mock_response['data']['available']
+    assert response.outputs[0]['status'] == mock_response['data']['status']
+    assert response.outputs[0]['unlock_code'] == mock_response['data']['unlock_code']
+
+
+def test_event_list_command(requests_mock, mock_client):
+    """
+    Scenario:
+    -   Get list of events.
+    Given:
+    -   The user has entered no arguments.
+    When:
+    -    cisco-amp-event-list is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure there are no links in the outputs.
+    """
+    mock_response = load_mock_response('event_list_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/events',
+        json=mock_response
+    )
+
+    args: Dict[str, Any] = {}
+
+    from CiscoAMP import event_list_command
+    responses = event_list_command(mock_client, args)
+
+    # Not including last element as it is only a readable output.
+    for response in responses[:-1]:
+        assert response.outputs_prefix == 'CiscoAMP.Event'
+
+        if 'file' in response.outputs:
+            assert response.indicator.sha256 == response.outputs['file']['identity']['sha256']
+            assert response.indicator.path == response.outputs['file']['file_path']
+            assert response.indicator.name == response.outputs['file']['file_name']
+
+        if computer := response.outputs.get('computer'):
+            assert 'links' not in computer
+
+
+@pytest.mark.parametrize(
+    'args, expected_number_of_results, expected_value',
+    [({}, 100, 'data[0]_id'),
+     ({'limit': '50'}, 50, 'data[0]_id'),
+     ({'page': '7', 'page_size': '5'}, 5, 'data[30]_id')]
+)
+def test_event_types_list_command(requests_mock, mock_client, args, expected_number_of_results, expected_value):
+    """
+    Scenario:
+    -   Get list of event types.
+    Given:
+    -   The user has entered no arguments.
+    -   The user has entered automatic pagination.
+    -   The user has entered manual pagination.
+    When:
+    -    cisco-amp-event-type-list is called.
+    Then:
+    -   Ensure outputs_prefix is correct.
+    -   Ensure pagination has worked.
+    """
+    mock_response = load_mock_response('event_type_list_response.json')
+    requests_mock.get(
+        f'{BASE_URL}/event_types',
+        json=mock_response
+    )
+
+    from CiscoAMP import event_type_list_command
+    response = event_type_list_command(mock_client, args)
+
+    assert response.outputs_prefix == 'CiscoAMP.EventType'
+    assert len(response.outputs) == expected_number_of_results
+    assert response.outputs[0]['id'] == expected_value
```

Comment:
Please compare the whole output, not just the `id`.

Reply:
Changed.